src/main.js

```diff
@@ -214,27 +213,28 @@ function savePosterToGrid() {
   savePoster(currentPoster);
 }
 
-function goToMain() {
-  hideForm();
-  showMainPoster();
-  hideSavedPosters();
-}
-//refactor remove/add to toggle?
-function showForm() {
-  form.classList.remove("hidden");
+function deletePoster() {
+  //poster will be deleted from grid
+  //target ---> savedPostersGrid
+  //target class ---> "mini-poster"
+  if (event.target.classList.contains("mini-poster")) {
```

Comment:
As far as the issue with the clickable area only being on the edge is here. Right now we are only concerned with the target that has the class name of `mini-poster`. But if you look around with your element inspector, you can see that everything within the `mini-poster` section has a different class name or an id instead. I would look into how to target children elements!

Reply:
That makes sense-- I was using the dev tools and thought that I had to target everything with "mini-poster" since they all seemed to have that same class name attached but I was thinking about how to target everything inside, so I'll look into target the children.  Thank you!